app/controllers/json_api/external_invoices_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of hitobito and
+#  licensed under the Affero General Public License version 3 or later. See the
+#  COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class JsonApi::ExternalInvoicesController < JsonApiController
+  def index
+    authorize!(:index, ExternalInvoice)
+    super
+  end
+
+  def show
+    authorize!(:show, entry)
+    super
+  end
+
+  def update
+    authorize!(:update, entry)
+    super
+  end
+
+  private
+
+  def entry
+    @entry ||= ExternalInvoice.find(params[:id])
```

Comment:
Imho wäre es besser, hier den `accessible_by` scope zu verwenden, den die Ability bietet. 
Die Fehlermeldung ist dann mit 404 zwar weniger sprechend, dafür haben wir kein loch falls der check beim update oder show mal wegfallen sollte, imho sollte das falls möglich sogar in der ApplicationAbility passieren